project/LibDependencies.scala

```diff
@@ -24,6 +24,7 @@ object LibDependencies {
     "com.typesafe"            % "config"          % "1.4.1",
     "com.beachape"           %% "enumeratum"      % "1.7.0",
     "nu.validator"            % "validator"       % "20.7.2",
+    "com.typesafe.play"      %% "twirl-api"       % "1.5.1"   % "test",
```

Comment:
Would this work as it's only scoped to test? We did discuss not bringing in Play as a dependency as we would then need to start thinking about cross-compilation.

Reply:
yeah I'm not sure, that's part of what I'm curious about here, I feel like I don't have a good nose for if this is a bit of a bad smell when working on a scala library or a reasonable way to approach it. I guess this PR is more a "don't do that because _" mad-lib than should we actually do it